[W12-1] Lost and Found

plty opened the pull request:
@dan-delion 
@Swiftkage 
@kevvvinn

plty committed:
change repo badges source from se-edu into our organization

Commit:
Merge pull request #3 from CS2103-AY1819S1-W12-1/readme-edit

change repo badges source from se-edu into our organization

plty committed:
add encrypted slack notification token

plty committed:
reformat travis.yml file

plty committed:
add unencrypted slack notification token

plty committed:
remove team-members placeholder images

plty committed:
add team-members placeholder photos

plty committed:
change AboutUs.adoc template

Commit:
Merge pull request #4 from CS2103-AY1819S1-W12-1/travis-ci-slack-integration

Travis CI Slack Integration

plty committed:
Merge branch 'master' of github.com:nus-cs2103-AY1819S1/addressbook-level4 into dev

plty committed:
Merge branch 'master' into dev

plty committed:
move coveralls link from address-book to main

plty committed:
Merge branch 'coveralls-fix' into dev

plty committed:
move appveyor link from address-book to main

Commit:
Merge pull request #8 from CS2103-AY1819S1-W12-1/dev

repo sync with upstream, coverrals and appveyor fix

Commit:
Update badge in README to new repo name

plty committed:
remove old codacy badge

Commit:
Update codacy source

Commit:
Update badge link

plty committed:
change travis badge image source

plty committed:
update wrong travis image source change

Commit:
Merge pull request #11 from CS2103-AY1819S1-W12-1/readme/badge

Update badge in README to new repo name

plty committed:
Merge pull request #12 from nus-cs2103-AY1819S1/master

DeveloperGuide: update "Setting up" section

plty committed:
add swiftkage and dan-delion photo and template for kevvvinn

plty committed:
moving the photo link to the respective file

plty committed:
Merge branch 'documentation' of github.com:plty/nuspro-lost-and-found into documentation

plty committed:
remove learning outcomes LearningOutcomes.adoc

plty committed:
amend readme, change readme into lost and found

plty committed:
User Guide Changes

plty committed:
change developer guide

plty committed:
Merge pull request #16 from plty/documentation

Documentation

Commit:
Merge pull request #1 from CS2103-AY1819S1-W12-1/master

asdf

Commit:
docs/images: change Ui.png

Swiftkage committed:
Edit photo location

Commit:
Merge pull request #21 from Swiftkage/images

Edit photo location

plty committed:
Merge branch 'master' into master

Commit:
Merge pull request #17 from dan-delion/master

Change Ui.png UI mockup in docs/images

Swiftkage committed:
Merge pull request #1 from CS2103-AY1819S1-W12-1/master

Update

Swiftkage committed:
Modify find command

Command find-(FIELD) general implementation.
Implemented for name field
Changed OR to AND operation

Swiftkage committed:
Modify coding style

asiddharth commented on README.adoc:
```diff
@@ -1,40 +1,25 @@
-** A more sophisticated GUI that includes a list  panel and an in-built Browser.
-** More test cases, including automated GUI testing.
-** Support for _Build Automation_ using Gradle and for _Continuous Integration_ using Travis CI.
+* This is a desktop lost and found application. It has a GUI but most of the user interactions happen using a CLI (Command Line Interface).
```
Maybe add a screen shot of your application to your readme file.

asiddharth commented on README.adoc:
```diff
@@ -1,40 +1,25 @@
 
-* Some parts of this sample application were inspired by the excellent http://code.makery.ch/library/javafx-8-tutorial/[Java FX tutorial] by
-_Marco Jakob_.
+* This application is based on se-edu https://github.com/se-edu/addressbook-level4/[Address Book Level 4]
```
Good job on the acknowledgement!

asiddharth commented on docs/DeveloperGuide.adoc:
```diff
@@ -836,63 +836,158 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+* 1a. The command format is invalid.
++
+[none]
+** 1b. System sends error message and command instruction to user.
```
Should be 1a1 and not 1b

asiddharth commented on docs/DeveloperGuide.adoc:
```diff
@@ -836,63 +836,158 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+* 2a. The list is empty
 +
 Use case ends.
+* 2b. The given index is invalid.
```
Should this not be 3a ? Does the system give an error message?

asiddharth commented on docs/DeveloperGuide.adoc:
```diff
@@ -836,63 +836,158 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
 [none]
-** 3a1. AddressBook shows an error message.
+* 3a. Invalid command format
```
There is no step 3 in MSS. The extension should be 1a ?

asiddharth commented on docs/DeveloperGuide.adoc:
```diff
@@ -836,63 +836,158 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+*Extensions*
+
+[none]
+* 4a. The list is empty
```
2a?

asiddharth commented on docs/DeveloperGuide.adoc:
```diff
@@ -836,63 +836,158 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+* 4a. The list is empty
++
+Use case ends.
+* 4b. The given index is invalid.
```
3a?

asiddharth commented on docs/DeveloperGuide.adoc:
```diff
@@ -836,63 +836,158 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+*Extensions*
+
+[none]
+* 5a. The list is empty
```
The same comment. Should be 2a.

asiddharth commented on docs/DeveloperGuide.adoc:
```diff
@@ -836,63 +836,158 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+* 5a. The list is empty
++
+Use case ends.
+* 5b. The given index is invalid.
```
Should be 3a

asiddharth commented on docs/DeveloperGuide.adoc:
```diff
@@ -836,63 +836,158 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 .  Should work on any <<mainstream-os,mainstream OS>> as long as it has Java `9` or higher installed.
-.  Should be able to hold up to 1000 persons without a noticeable sluggishness in performance for typical usage.
+.  Should be able to hold up to 1000 Items without a noticeable sluggishness in performance for typical usage.
 .  A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
```
Maybe good to add more NFRs.

asiddharth replied:
Same comment again.

Swiftkage committed:
Modify test cases

Modify test cases for find command

asiddharth reviewed (commented):
Added some comments regarding the use cases. Please acknowledge that you have read them.

Swiftkage committed:
Update checkstyle

Commit:
Change window layout

Commit:
Update README.adoc

plty committed:
Merge pull request #24 from Swiftkage/find

Modify Find command

plty committed:
Merge pull request #26 from CS2103-AY1819S1-W12-1/dan-delion-patch-1

Update README.adoc

Commit:
Merge pull request #2 from CS2103-AY1819S1-W12-1/master

Merge from upstream

plty committed:
Merge pull request #25 from dan-delion/ui-skeleton

Change window layout

plty committed:
Merge branch 'master' into develop/ui-skeleton

plty committed:
Merge branch 'master' into develop/find

Commit:
Merge pull request #28 from CS2103-AY1819S1-W12-1/develop/ui-skeleton

Develop/ui skeleton

Commit:
Merge pull request #27 from CS2103-AY1819S1-W12-1/develop/find

Develop/find

plty committed:
move from person to article

Commit:
Merge pull request #29 from plty/move-to-articles

move from person to article

Commit:
Update UserGuide.adoc

plty committed:
Merge pull request #30 from dan-delion/master

Update UserGuide.adoc

Commit:
Merge pull request #3 from CS2103-AY1819S1-W12-1/master

Merge from upstream

Commit:
Update ContactUs.adoc

Commit:
Update AboutUs.adoc

Commit:
Update UserGuide.adoc

Fixed typos, removed some questionable features.

Commit:
Merge pull request #4 from CS2103-AY1819S1-W12-1/master

Update docs

asiddharth commented on README.adoc:
```diff
@@ -15,26 +14,20 @@ ifndef::env-github[]
-** More test cases, including automated GUI testing.
-** Support for _Build Automation_ using Gradle and for _Continuous Integration_ using Travis CI.
+* This is a desktop lost and found application. It has a GUI but most of the user interactions happen using a CLI (Command Line Interface).
+* This application is made for lost and found administrators who are comfortable with CLI.
```
You might want to add some more information about your product.

asiddharth commented on docs/AboutUs.adoc:
```diff
@@ -4,53 +4,33 @@
-Role: Project Advisor
+=== Bobbie Ramadhani Soedirgo
+image::dan-delion.png[width="150", align="left"]
+{empty}[https://github.com/dan-delion[github]] [<<dan-delion#, portfolio>>]
```
May be helpful to add the portfolio for each team member. Will be useful to keep everything updated as you progress with the project.

https://nus-cs2103-ay1819s1.github.io/cs2103-website/admin/project-deliverables.html#deliverable-project-portfolio-page-ppp

asiddharth commented on README.adoc:
```diff
@@ -1,11 +1,10 @@
+https://travis-ci.com/CS2103-AY1819S1-W12-1/main[image:https://travis-ci.com/CS2103-AY1819S1-W12-1/main.svg?branch=master[Build Status]]
+https://ci.appveyor.com/project/CS2103-AY1819S1-W12-1/main/branch/master[image:https://ci.appveyor.com/api/projects/status/yvil727ad656xo0d/branch/master?svg=true[Build status]]
+https://coveralls.io/github/CS2103-AY1819S1-W12-1/main?branch=master[image:https://coveralls.io/repos/github/CS2103-AY1819S1-W12-1/main/badge.svg?branch=master[Coverage Status]]
+https://www.codacy.com/app/CS2103-AY1819S1-W12-1/main?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=CS2103-AY1819S1-W12-1/main&amp;utm_campaign=Badge_Grade[image:https://api.codacy.com/project/badge/Grade/8659d1e101a3438cb1fdcff23080c3b4[Codacy Badge]]
```
Good job on :

Adding UI Mockup
Acknowledging AB4
Adding correct links for travis.

asiddharth commented on docs/ContactUs.adoc:
```diff
@@ -2,6 +2,6 @@
+* *Bug reports, Suggestions* : Post in our https://github.com/CS2103-AY1819S1-W12-1/main/issues[issue tracker] if you noticed bugs or have suggestions on how to improve.
 * *Contributing* : We welcome pull requests. Follow the process described https://github.com/oss-generic/process[here]
-* *Email us* : You can also reach us at `damith [at] comp.nus.edu.sg`
+* *Email us* : You can also reach us at `sram-b [at] comp.nus.edu.sg`
```
Good job here!!

asiddharth commented on docs/UserGuide.adoc:
```diff
@@ -1,30 +1,32 @@
 == Introduction
 
-AddressBook Level 4 (AB4) is for those who *prefer to use a desktop app for managing contacts*. More importantly, AB4 is *optimized for those who prefer to work with a Command Line Interface* (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB4 can get your contact management tasks done faster than traditional GUI apps. Interested? Jump to the <<Qui ...
+Lost and Found is made for lost and found (duh) administrators... +
```
Nice intro!

asiddharth commented on docs/UserGuide.adoc:
```diff
@@ -1,30 +1,32 @@
-.  Download the latest `addressbook.jar` link:{repoURL}/releases[here].
-.  Copy the file to the folder you want to use as the home folder for your Address Book.
+.  Ensure you have Java version `9` or later installed in your computer.
+.  Download the latest `Lost and Found.jar` link:{repoURL}/releases[here].
```
Good job here!

asiddharth commented on docs/UserGuide.adoc:
```diff
@@ -46,211 +47,119 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
+e.g. `edit 2 n/Jacket t/leather`
 * *Find* : `find KEYWORD [MORE_KEYWORDS]` +
-e.g. `find James Jake`
+e.g. `find brown wallet`
```
Might want to add examples for your command (ex : resolve)

asiddharth commented on src/main/java/seedu/address/logic/parser/FindCommandParser.java:
```diff
@@ -26,8 +33,15 @@ public FindCommand parse(String args) throws ParseException {
+        List<String> listKeywords = new LinkedList<>(Arrays.asList(nameKeywords));
 
-        return new FindCommand(new NameContainsKeywordsPredicate(Arrays.asList(nameKeywords)));
+        if (listKeywords.get(0).equals("-n")) { //different operations depending on FIELD
```
Comments should not be written like this. Please refer to coding standards

asiddharth commented on docs/DeveloperGuide.adoc:
```diff
@@ -836,63 +836,158 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+|`* *`   |user |Redo the last n actions|Cancel undo actions that I did by mistake
+
+|`* *`   |user |Add / remove multiple tags from an item |
```
"So that" part missing

asiddharth reviewed (commented):
Good job overall. Here are some comments that may help you.

1. The link seems broken 
![broken](https://user-images.githubusercontent.com/23400124/46577896-e5086980-ca23-11e8-9fb9-d0dee0bd5ec1.png)

2. Might want to rectify failed builds.
![fails_build](https://user-images.githubusercontent.com/23400124/46577923-65c76580-ca24-11e8-97d8-6f62142ec918.png)
3. Feature 3.4 title is wrong
![3 4 _missing](https://user-images.githubusercontent.com/23400124/46578048-58f84100-ca27-11e8-82ec-1bb969acf583.png)

4. Update the product scope/value proposition.
![13-2_dg1](https://user-images.githubusercontent.com/23400124/46578093-2ef34e80-ca28-11e8-88f9-985c6369aebe.png)

Commit:
Fix checkstyle issues.

Commit:
Update docs/team

Commit:
Fix checkstyleTest issues

Swiftkage committed:
Merge pull request #36 from dan-delion/master

Fix checkstyle issues causing tests to fail

Swiftkage committed:
Merge pull request #5 from CS2103-AY1819S1-W12-1/master

Update changes

Commit:
Minor fxml changes

Commit:
Add ArticleDetailsPanel

Commit:
Add ArticleDetailsPanelHandle

Commit:
Add ArticleDetailsPanelTest

Commit:
Update MainWindowHandle

Commit:
Update AddressBookSystemTest

Commit:
Update AddressBookSystemTest and HelpWindowSystemTest

Commit:
Remove BrowserPanel functionality completely

Commit:
Fix nonfunctional errors

Commit:
Rename ArticleListCard.fxml to ArticleCard.fxml

Commit:
Add newline at eof of CommandBox.fxml

Commit:
Merge pull request #40 from dan-delion/master

Merge UI skeleton updates

Swiftkage committed:
Add resolve

Add resolve command.
Currently deletes only
Teast cases not created yet

Swiftkage committed:
Update resolve

Swiftkage committed:
Merge pull request #7 from Swiftkage/master

Update

Swiftkage committed:
Update resolve

Swiftkage committed:
Merge pull request #41 from Swiftkage/resolve

add Resolve

Commit:
Add GridPane to article details

Commit:
Add placeholder photo in article details

Commit:
Add placeholder photo in article details

Commit:
Merge branch 'master' of github.com:dan-delion/main

Commit:
Add description

Commit:
Merge pull request #42 from dan-delion/master

Add placeholder photo

Swiftkage committed:
Add isResolved field

Initialize isResolved to false

Swiftkage committed:
Merge pull request #10 from Swiftkage/master

Add isResolved field

plty committed:
change all instances of addressbook into articlelist

plty committed:
renaming file from addressbook to articlelist

plty committed:
move from seedu.address to seedu.lostandfound

plty committed:
change addressbook.jar into articlelist.jar

plty committed:
change order of import to comply with checkstyle

plty committed:
change from address to description by script

plty committed:
move back description_book_32.png into address_book_32.png

plty committed:
edit to comply with checkstyle

plty committed:
Merge pull request #55 from CS2103-AY1819S1-W12-1/move-address-to-description

move AddressBook into ArticleList, seedu.address into seedu.lostandfound, Address into Description

Commit:
Merge pull request #56 from dan-delion/master

Merge

plty committed:
resolve by git auto-merge

plty committed:
move CommandException old link to new one

plty committed:
change from address to description

plty committed:
remove lol.out

plty committed:
resolve commit conflicts

plty committed:
remove trailing space

Commit:
Merge pull request #57 from CS2103-AY1819S1-W12-1/ui-skeleton-merge

Ui skeleton merge

Commit:
Add name field in finder and owner panels

plty committed:
simplify validation regex and message constraints

Commit:
Fix divider

Commit:
Remove grid lines

Commit:
merge to master

Commit:
Make fxml great again

Commit:
Make fxml great again again

Commit:
Make fxml great again again again

Swiftkage committed:
Merge branch 'master' into simplify-persons-attribute

Swiftkage committed:
Merge pull request #59 from CS2103-AY1819S1-W12-1/simplify-persons-attribute

simplify validation regex and message constraints

Swiftkage committed:
Update resolve

Added a field isResolved in Article class to differentiate resolved objects.

All Acticle objects isResolved field is set to false initially, resolve will set the field to true.

Swiftkage committed:
Update resolve

Resolve command will set isResolved field to true.

Test cases not added yet.

Swiftkage committed:
update checkstyle

Swiftkage committed:
Try to resolve merge conflict

Swiftkage committed:
Revert "Try to resolve merge conflict"

This reverts commit 039870a0f963d3b4b2f27c569139c435a9914f14.

Swiftkage committed:
Merge pull request #61 from Swiftkage/resolve

Update resolve

Swiftkage committed:
Merge branch 'master' of https://github.com/CS2103-AY1819S1-W12-1/main into CS2103-AY1819S1-W12-1-master

Swiftkage committed:
Fix checkstyle

Swiftkage committed:
Merge pull request #63 from Swiftkage/master

Update from upstream

plty committed:
Merge pull request #64 from CS2103-AY1819S1-W12-1/develop/resolve

Add resolve command

Swiftkage committed:
Update find command

Added findall, findresolved and find-d command

findall: matches keyword to all fields, OR operation

findresolve: matches keyword to all fields which isResolved == true. OR operation

find-d: matches keyword to Description field. AND operation

Swiftkage committed:
Fix checkstyle

plty committed:
change typo of Descriptiones into Descriptions

plty committed:
Merge pull request #66 from CS2103-AY1819S1-W12-1/remove-typo

change typo of Descriptiones into Descriptions

plty committed:
refactor and generalize

plty committed:
add sequence class

plty committed:
add Image class

asiddharth reviewed (commented):
Hi team,! Here are some suggestions :
1. 
![w12-1](https://user-images.githubusercontent.com/23400124/47268052-5c8adc80-d57e-11e8-843e-c6805975b1d0.png)
(a) @Swiftkage Face is too small in the photo. Please update it.
(b) @kevvvinn Please upload your photo.
(c) The build should be passing. Saw the travis page, and there are no builds. Just one cancelled build. 
Seems you are using the travis-ci.com instead of travis-ci.org. Please update the links. You can find the links at https://nus-cs2103-ay1819s1.github.io/cs2103-website/admin/projectList.html 

Suggestions for DG :
Your DG looks very similar to that of the parent repository. You might want to consider doing some of the following things.
1. You might want to add some architecure, sequence and class diagrams tailored to your project application.
2. You might want to add some feature implementation details.
3. Add more NFRs.

There is a lot to do. Good luck!

Commit:
Update README.adoc

Commit:
Merge branch 'master' of github.com:CS2103-AY1819S1-W12-1/main

Commit:
Make fxml great again again again again

Swiftkage committed:
Update filtered list

Commands: list, add, redo, undo, resolve, edit will return list of unresolved articles by default.

Swiftkage committed:
Merge pull request #14 from CS2103-AY1819S1-W12-1/master

Update

Swiftkage committed:
Update swiftkage.png

plty committed:
Merge pull request #68 from Swiftkage/master

So many commits

plty committed:
update dg references from addressbook to articlelist and so on

Swiftkage committed:
Merge pull request #69 from CS2103-AY1819S1-W12-1/docs/dg-update

update dg references from addressbook to articlelist and so on

Swiftkage committed:
Update developer guide

plty committed:
Merge branch 'master' into develop/find

plty committed:
Merge pull request #70 from CS2103-AY1819S1-W12-1/develop/find

Update find and filtered list

Swiftkage committed:
Merge pull request #71 from Swiftkage/master

Update DG

Swiftkage committed:
Merge branch 'master' into docs/dg-update

plty committed:
Merge pull request #72 from CS2103-AY1819S1-W12-1/docs/dg-update

Update DG

plty committed:
merge master

plty committed:
make path at image stored as an object instead of string

plty committed:
remove id value bug

plty committed:
change accessor type of Image class

plty committed:
add copy command in FileUtil

plty committed:
changing signature

plty committed:
change sequence return type to Integer

plty committed:
add Image to constructor of Article

plty committed:
adding throws and test data

plty committed:
change placehoder into real image

plty committed:
default image of an item

plty committed:
test photo

plty committed:
image create filename bug

Swiftkage committed:
Update UG and Article

Swiftkage committed:
Update UG

Swiftkage committed:
Merge pull request #15 from CS2103-AY1819S1-W12-1/master

Merge

Swiftkage committed:
Update DG

Swiftkage committed:
Update Article

This line is failing the tests.... will fix next time once all parameters are finalized

Swiftkage committed:
Update UG

Swiftkage committed:
Merge pull request #74 from Swiftkage/master

Update UGDG

plty committed:
dunno

plty committed:
make image editable

plty committed:
Merge pull request #75 from CS2103-AY1819S1-W12-1/yuxi-patch

Update UGDG

plty committed:
merging with master

Swiftkage committed:
Add FINDER and OWNER fields

Swiftkage committed:
Update PPP, UG, DG

Swiftkage committed:
Update DG

Swiftkage committed:
Update test cases

Swiftkage committed:
Update AboutUs

Swiftkage committed:
Merge branch 'patch'

Swiftkage committed:
Merge branch 'master' into master

Swiftkage committed:
Update build.gradle

plty committed:
Merge pull request #80 from Swiftkage/master

Update so many things

Swiftkage committed:
Update UG and find

Swiftkage committed:
Update DG

Swiftkage committed:
Merge pull request #16 from CS2103-AY1819S1-W12-1/master

Merge pull request #80 from Swiftkage/master

Swiftkage committed:
Merge pull request #122 from Swiftkage/master

Update UGDG and resolve

plty committed:
bugfix on Image testcases and handle null image at xml loading

plty committed:
merge with master

Swiftkage committed:
Rename Swiftkage.adoc to swiftkage.adoc

Swiftkage committed:
Rename Swiftkage.adoc to swiftkage.adoc

Commit:
Change default image, fix app name

Commit:
Add finder fields Phone and Email, onto the finder panel.

Commit:
Remove phone and email from article cards

Commit:
Wrap article card description.

Commit:
Better UI

asiddharth reviewed (commented):
Some comments : 
1. Sequence diagrams for Find and Resolve features seem to be missing. 
![find_seq_diag_missing](https://user-images.githubusercontent.com/23400124/48132601-287a2000-e2cf-11e8-91de-951f8a194b25.png)
![resolve_seq_diag_missing](https://user-images.githubusercontent.com/23400124/48132616-2ca63d80-e2cf-11e8-93b7-5e129c3eb8cf.png)
2. The details about the feature that attaches images are not mentioned in the DG. 
A seq diagram would also help along with this.

3. Sections such as "Appendix A: Suggested Programming Tasks to Get Started " do not seem relevant to your product. Please remove such unrelated sections from the DG.

4. Can add activity diagrams for use cases. 
![activity](https://user-images.githubusercontent.com/23400124/48132994-45fbb980-e2d0-11e8-91bd-78abd5b93288.png)

5.  Also recommended to add architecture diagrams for any major components added.

6. Would be good to add more test cases. If not remove "{ more test cases …​ }"
![more_test_cases](https://user-images.githubusercontent.com/23400124/48133088-9b37cb00-e2d0-11e8-915d-c2fc57cf66f7.png)

Good luck with the demo!

asiddharth reviewed (commented):
Several references to AB4 still exist in the DG. Remove the ones which are irrelevant.

Swiftkage committed:
Merge pull request #124 from CS2103-AY1819S1-W12-1/develop/default-image

Change default image, fix app name

Swiftkage committed:
Merge pull request #125 from CS2103-AY1819S1-W12-1/develop/add-finder-fields

Develop/add finder fields

Swiftkage committed:
Merge pull request #126 from CS2103-AY1819S1-W12-1/develop/ui-enhancements

Develop/ui enhancements

plty committed:
i have to go home

plty committed:
fix xml test

Swiftkage committed:
Merge pull request #18 from CS2103-AY1819S1-W12-1/master

Update from upstream

Swiftkage committed:
Update DG and test cases

Swiftkage committed:
Upate project specs

Swiftkage committed:
Update UI

Swiftkage committed:
Merge pull request #129 from Swiftkage/master

Update project specs

Swiftkage committed:
Update test case data

Swiftkage committed:
Change Article Identity fields

And also relevant test cases

Swiftkage committed:
Update DG

Swiftkage committed:
Update DG

Swiftkage committed:
Update Swiftkage PPP

Swiftkage committed:
Merge pull request #130 from Swiftkage/master

Update DG and PPP

plty committed:
xml parser bug fix by setting image to optional

Swiftkage committed:
Update DG, PPP and find-t

plty committed:
checktyle related updates

Swiftkage committed:
Update test cases

Added test data for resolved articles.
Made sure filteredList test are working.

Swiftkage committed:
Merge pull request #131 from Swiftkage/master

Update DG, PPP and find-t

Swiftkage committed:
Add test cases for Edit

Swiftkage committed:
Fix checkstyle

Swiftkage committed:
Update Add and Edit test cases

plty committed:
merge with master

plty committed:
resolve conflict with newest merge

plty committed:
Merge branch 'image-integration'

plty committed:
Merge pull request #132 from Swiftkage/master

Update test cases

Swiftkage committed:
Add test cases for predicates

Swiftkage committed:
Update AboutUs

plty committed:
first draft

Swiftkage committed:
Fix checkstyle

Swiftkage committed:
Add predicate test cases

Commit:
Update dan-delion.adoc

Swiftkage committed:
Merge pull request #135 from Swiftkage/master

Add test cases for predicates

plty committed:
jerrell PPP

plty committed:
Merge pull request #136 from CS2103-AY1819S1-W12-1/ppp-jerrell

Ppp jerrell

Swiftkage committed:
Update swiftkage.adoc

Swiftkage committed:
Update swiftkage.adoc
